src/Pagination.tsx

```diff
@@ -35,6 +37,10 @@ export default function Pagination({
 
   let renderedElements = null;
 
+  if(!alwaysVisible && maxPage === 1) {
+    return null;
+  }
+
```

Comment:
Hmm, since we handle the conditional rendering in the `Table.tsx` file, I don't think this is needed? Or is there something that I am missing?

Reply:
You are right, I guess this is where I started and then moved the checks back to Table, so no need for this extra prop or check. Good catch